src/site/content/en/blog/youtube-media-capabilities/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: YouTube enhances UX with Media Capabilities API
+subhead: |
+  Ensure buttery smooth playback when your users are on different devices.
+authors:
+  - beaufortfrancois
+date: 2019-06-05
+hero: hero.jpeg
+hero_position: top
+alt: Youtube application screengrab
+description: |
+  In an experiment with the Media Capabilities API, YouTube saw a 7.1% increase
+  in MTBR with only a 0.4% decrease in average resolution of videos served.
+tags:
+  - post
+  - media
+  - ux
+  - case-study
+---
+
+In an experiment with the Media Capabilities API, YouTube saw a 7.1% increase in
+MTBR with only a 0.4% decrease in average resolution of videos served.
+
+{% Aside 'key-term' %}
+MTBR (Mean Time Between Rebuffers) is total play time divided by the number of
+rebuffering events.
+{% endAside %}
+
+## The Problem
+
+Typically, media sites will have several variants that they can present users,
+encoded in different frame rates, resolutions, and codecs. Until recently, while
+choosing which variant to play for an individual user, web developers had to
+rely solely on `isTypeSupported()` or `canPlayType()` to determine whether each
+variant could be played in that user’s browser. While this told the developer
+whether media could be played at all, it didn’t provide an indication of
+playback quality, such as whether the media playback would experience frame
```

Comment:
```suggestion
playback quality, such as whether there would be frame
```
For concision and to avoid misattribution of the verb. (The user experiences things, not the media playback.)

Reply:
Done.